scripts/prune_workers.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+set -e
```

Comment:
no need to set -e, if you already have bash -e 2 lines above

Reply:
Yet another overlooking mistake :)